Lib/shutil.py

```diff
@@ -105,7 +105,14 @@ def copymode(src, dst):
         os.chmod(dst, mode)
 
 def copystat(src, dst):
-    """Copy all stat info (mode bits, atime, mtime, flags) from src to dst"""
+    """Copy file metadata
+
+    Copy the permission bits, last access time, last modification time, and
+    flags from `src` to `dst`. On Linux, copystat() also copies the "extended
+    attributes" where possible. The file contents, owner, and group are
+    unaffected. `src` and `dst` are path names given as strings.
+
```

Comment:
nitpick: please remove the empty line

Reply:
"The reason why there was a newline is that almos 99% of the functions are documented like this, by having an extra line at the end of the docstring."

Well... Python has a long history. Python has been created before PEP 7, PEP 8, PEP 257, etc. I don't see an advice to insert an empty line at the end of docstring in the docstring convention PEP: https://www.python.org/dev/peps/pep-0257/

cc @ambv: any progress on your secret project to convert Python to Black? ;-)